src/reports/biospecimen-data/index.ts

```diff
@@ -0,0 +1,36 @@
+import { Request, Response } from 'express';
+import { Client } from '@elastic/elasticsearch';
+import generateReport from '../generateReport';
+import reportConfig from './config';
+import { normalizeConfigs } from '../../utils/configUtils';
```

Comment:
I think you could get rid of ` ../../ `

Reply:
Not sure to understand your comment. This is the correct path to access to configUtils from this file.
If I remove them, the import can't be resolved anymore. Even if I change typescript config to add src base url, so typescript will let me do : `import { normalizeConfigs } from 'utils/configUtils';`
It will break at runtime because node resolves import that doesn't start by . or .. like it's a dependancy which is not the case.
Eventually, we could add webpack to solve this issue (not even sure) but I think this is overkill in comparison to the benefit

Reply:
I viewed it as a matter of choosing a convention between absolute (to the src) vs relative path. However, your arguments are fine. It's all good on my side.

Reply:
you can't use `import { normalizeConfigs } from 'src/utils/configUtils';`
because node will try to find the module in external dependancies because if path doesn't start with . or .. it will not be considered as local dependancy